recompose-idea/src/main/kotlin/recompose/plugin/copypaste/ConvertXmlToComposeConfirmationDialog.kt

```diff
@@ -34,6 +36,9 @@ class ConvertXmlToComposeConfirmationDialog(project: Project) : DialogWrapper(pr
     @Suppress("unused")
     lateinit var questionLabel: JLabel
 
+    @Suppress("unused")
+    lateinit var donTShowThisCheckBox: JCheckBox
```

Comment:
nit: `donT` should probably be `dont` or maybe `doNot`?

Reply:
I've changed that to dontShowThisCheckBox